robottelo/ui/user.py

```diff
@@ -91,23 +101,15 @@ def create(self, username=None, email=None, password1=None,
                 )
         self.click(common_locators['submit'])
 
-    def search(self, name, search_key):
-        """Searches existing user from UI."""
-        Navigator(self.browser).go_to_users()
-        return self.search_entity(
-            name, locators['users.user'], search_key=search_key)
-
-    def delete(self, name, search_key, really=True):
+    def delete(self, name, really=True):
         """Deletes existing user from UI."""
         self.delete_entity(
             name,
             really,
-            locators['users.user'],
```

Comment:
I really have no idea how it was working before.
We were passing user locator as locator to 'delete' button, and locator to delete button as locator to drop-down (and there's no drop down, actually). Of course it was not working locally. What surprises me is the fact it wasn't failing on Jenkins.